app/components/specific/cards/AthletePerformancesCard.js

```diff
@@ -5,29 +5,31 @@ import CardList from "../../common/cardList/CardList";
 import { getIconName } from "../../../routes/main/clubFeed/helper";
 import theme from "../../../theme/theme";
 
-export default class AthleteDetailsCard extends Component {
-  // ToDO fix shape
+export default class AthletePerformancesCard extends Component {
   static propTypes = {
     rendered: PropTypes.bool,
-    athlete: PropTypes.shape({
-      firstname: PropTypes.string,
-      lastname: PropTypes.string,
-      profil: PropTypes.string
-    }).isRequired
+    performances: PropTypes.arrayOf(
+      PropTypes.shape({
+        name: PropTypes.string,
+        percent: PropTypes.number,
+        value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
string?

Reply:
Yep! Don't remember why we did that but it could be one of this type.

Reply:
there is something fishy

Reply:
I need to check athlete helper just to be sure!